mindocr/postprocess/det_pse_postprocess.py

```diff
@@ -1,14 +1,45 @@
 import cv2
 import numpy as np
 
-import mindspore as ms
-from mindspore import Tensor, nn
+from mindspore import Tensor
 
 from .det_base_postprocess import DetBasePostprocess
 
 __all__ = ["PSEPostprocess"]
 
 
+def _resize_4d_array(pred, scale_factor):
+    """
+    Resize a 4D numpy array using bilinear interpolation in the H and W dimensions.
+    :param pred: A 4D numpy array of shape (N, C, H, W).
+    :param scale_factor: An integer value representing the scale factor applied to H and W.
+    :return: A 4D numpy array of shape (N, C, H', W'), where H' and W' are scaled versions of H and W.
+    """
+    N, C, H, W = pred.shape
+    scaled_H = H * scale_factor
+    scaled_W = W * scale_factor
+
+    # Reshape the 4D array into a 2D array
+    pred_2d = np.reshape(pred, (N * C, H, W))
+
+    # Resize the 2D array using cv2.resize
+    resized_2d = np.zeros((N * C, scaled_H, scaled_W))
+    for i in range(N * C):
+        resized_2d[i] = cv2.resize(pred_2d[i], (scaled_W, scaled_H), interpolation=cv2.INTER_LINEAR)
```

Comment:
As long as your image array have `(H, W, C*N)` shape, you can utilize OpenCV's batch processing instead of `for` loop:
https://stackoverflow.com/a/65160547/6380135
This should be much faster.

Reply:
Ok, try it now

Reply:
Modified. The FPS is improved from around 8 img/s to 10 now.